edpm_ansible/roles/edpm_tuned/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+# Copyright 2019 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# "{{ role_name }}" will search for and load any operating system variable file
+# found within the "vars/" path. If no OS files are found the task will skip.
+- name: Gather variables for each operating system
+  ansible.builtin.include_vars: "{{ item }}"
+  with_first_found:
+    - skip: true
+      files:
+        - "{{ ansible_facts['distribution'] | lower }}-{{ ansible_facts['distribution_version'] | lower }}.yml"
+        - "{{ ansible_facts['distribution'] | lower }}-{{ ansible_facts['distribution_major_version'] | lower }}.yml"
+        - "{{ ansible_facts['os_family'] | lower }}-{{ ansible_facts['distribution_major_version'] | lower }}.yml"
+        - "{{ ansible_facts['distribution'] | lower }}.yml"
+        - "{{ ansible_facts['os_family'] | lower }}-{{ ansible_facts['distribution_version'].split('.')[0] }}.yml"
+        - "{{ ansible_facts['os_family'] | lower }}.yml"
+  tags:
+    - always
+
+- name: Install packages
+  ansible.builtin.include_tasks: tuned_install.yml
+  when:
+    - (edpm_tuned_system_packages | length) > 0
+
+- name: Check for tuned-adm
+  ansible.builtin.command: >-
+    which tuned-adm
+  environment:
+    PATH: "/bin:/usr/bin:/sbin:/usr/sbin"
+  changed_when: false
+  failed_when: false
+  register: edpm_tuned_check
```

Comment:
i can't figure out why this task would be necessary, and if the check does fail, it seems like we'd want to error and exit, instead of silently skipping the configure tasks.